Admin-Component/react/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "vtex.admin-example",
+  "name": "vtexchile.admin-example",
```

Comment:
I think we need to define what vendor should we set in here, and update all packages.json accordingly

Reply:
No aplica